src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -643,3 +643,130 @@ Profile CascadiaSettings::_CreateDefaultProfile(const std::wstring_view name)
 
     return newProfile;
 }
+
+// Method Description:
+// - Gets our list of warnings we found during loading. These are things that we
+//   knew were bad when we called `_ValidateSettings` last.
+// Return Value:
+// - a reference to our list of warnings.
+std::vector<TerminalApp::SettingsLoadWarnings>& CascadiaSettings::GetLoadWarnings()

```

Comment:
```suggestion
std::vector<TerminalApp::SettingsLoadWarnings>& CascadiaSettings::GetWarnings()
```
Or are there any Settings related warnings that we may have at some point in the future. But even then, I feel like we'd want all of our warnings to just be in one list. No point in separating settings warnings by even more specific sub categories.